src/foundry/foundry.js/collections/documentCollections/worldCollections/users.d.ts

```diff
@@ -0,0 +1,42 @@
+import { ConfiguredDocumentClass } from '../../../../../types/helperTypes';
+
+declare global {
+  /**
+   * The singleton collection of User documents which exist within the active World.
+   * This Collection is accessible within the Game object as game.users.
+   *
+   * @see {@link User} The User entity
+   */
+  class Users extends WorldCollection<typeof foundry.documents.BaseUser, 'Users'> {
+    constructor(data?: foundry.data.UserData['_source'][]);
+
+    /**
+     * The User entity of the currently connected user
+     * @defaultValue `null`
+     */
+    current: InstanceType<ConfiguredDocumentClass<typeof foundry.documents.BaseUser>> | null;
+
+    /**
+     * Initialize the Map object and all its contained entities
+     */
+    protected _initialize(): void;
```

Comment:
Seems to be missing the input parameters.

Reply:
This is intentional, see https://gitlab.com/foundrynet/foundryvtt/-/issues/5571